smart_home/include/devices/ac_unit.h

```diff
@@ -0,0 +1,53 @@
+#ifndef SMART_HOME_DEVICES_AC_UNIT_H_
+#define SMART_HOME_DEVICES_AC_UNIT_H_
+
+#include "../rooms/room.h"
+#include "device.h"
+
+namespace devices {
+
+// Rule of Five
+
+class AcUnit : public Device {
+  public:
+    AcUnit(rooms::Room* room): Device(room), is_on_(false), fan_speed_(0) {}
```

Comment:
Although assigning default values in constructor is ok, I have a strong preference to default-initialise them where they are declared.
Also, only getters and setters are ok to be defined in header files. All other functions should be implemented in the class source file

eg. bool is_on{false}